src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -2,37 +2,43 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router';
 
-const Breadcrumbs = ((query, type) => {
-  const onClick = pageTitle => tracker('Breadcrumbs', pageTitle);
+/**
+ * Takes `query` and `type` as properties to pass to its methods.
+ *
+ * @param {object} props
+ * @returns {array}
+ */
+const Breadcrumbs = (({ links, pageType, onClickHandler }) => {
+  if (pageType === 'home') {
+    return null;
+  }
+
+  // const onClick = pageTitle => tracker('Breadcrumbs', pageTitle);
 
   const homeLink = (
     <li key="home">
-      <Link to={'/'} onClick={() => onClick('Home')}>
-        Home
+      <Link to="/" onClick={event => onClickHandler(event)}>
+        ResearchNow
       </Link>
     </li>);
 
   const crumbTrail = () => {
     const crumbs = [homeLink];
 
-    if (type === 'results') {
-      crumbs.push(<li key="results">Search Results</li>);
-    }
-
-    crumbs.push(
-      <li key="results">
-        <Link to={`/search?q=${query.q}`} onClick={() => onClick('Search Results')}>
-          Search Results
-        </Link>
-      </li>);
-
-    if (type === 'details') {
-      crumbs.push(<li key="details">Work Details</li>);
+    if (links && Array.isArray(links)) {
+      links.map((link, iterator) => {
+        if (iterator < links.length-1) {
+          crumbs.push(
+            <li key={iterator}>
+              <Link to={link.href}>
```

Comment:
Is it better to use the `href` property here instead of `to`?

Reply:
`to` is specific to `Link` so it's fine.